client/bindings/wasm/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@iota/iota-client-wasm",
+  "version": "0.5.0-alpha.6",
+  "description": "WebAssembly bindings for the IOTA client library",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iotaledger/iota.rs"
+  },
+  "author": "IOTA Foundation <contact@iota.org>",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "web/*",
+    "node/*",
+    "!web/test/",
+    "!node/test/"
+  ],
+  "scripts": {
+    "build": "npm run build:nodejs && npm run build:web",
+    "build:src": "cargo build --lib --release --target wasm32-unknown-unknown",
+    "build:nodejs": "node ./build_scripts/copyNeonDefs.js && npm run build:src && npm run bundle:nodejs && wasm-opt -O node/wasm/client_wasm_bg.wasm -o node/wasm/client_wasm_bg.wasm",
```

Comment:
What's the reason for having the `wasm-opt` part here and not in the Cargo.toml like this?
```toml
[package.metadata.wasm-pack.profile.release]
wasm-opt = true

[profile.release]
opt-level = 's'
```

Reply:
We stopped using `wasm-pack` and directly invoked `wasm-bindgen` because `wasm-pack` does not re-export the `--weak-refs` flag, which is necessary for proper garbage collection.

See also:
rustwasm/wasm-pack#930
iotaledger/identity.rs#694